frontend/components/commons/forms/questions-form/QuestionsForm.component.vue

```diff
@@ -28,6 +28,7 @@
           :title="input.question"
           :placeholder="input.placeholder"
           v-model="input.options[0].value"
+          :useMarkdown="input.use_markdown"
```

Comment:
with the settings attribute in ORM
```suggestion
          :useMarkdown="input.settings.use_markdown"
```